src/js/tabs/firmware_flasher.js

```diff
@@ -67,10 +67,16 @@ firmware_flasher.initialize = function (callback) {
             $('div.release_info #targetMCU').text(summary.mcu);
             $('div.release_info .configFilename').text(self.isConfigLocal ? self.configFilename : "[default]");
 
-            // Wiki link to "BOARD---" + target
+            // Wiki link to "BOARD " + target or just Wiki
             let targetWiki = $('#targetWikiInfoUrl');
-            targetWiki.html(`[Wiki: Board---${summary.target}]`);
-            targetWiki.attr("href", summary.wiki);
+            targetWiki.html(`&nbsp;&nbsp;&nbsp;[Wiki: Board ${summary.target}]`);
```

Comment:
I think we can probably just say "Wiki" - and not put the target in, as the target is already shown to the left

Reply:
Well, now changed as desired :-)